README.md

```diff
@@ -101,13 +101,13 @@ Additionally, romcal is also available for installation via various "release tag
 
 Require romcal in your project:
 
-```javascript
+```ts
```

Comment:
This should stay `javascript`, since examples in the documentation are made first for javascript.